src/api/sloppy.go

```diff
@@ -53,9 +57,7 @@ type Client struct {
 // NewClient returns a new Client to handle API requests.
 func NewClient() *Client {
 	client := &Client{
-		client: &http.Client{
-			Timeout: 2 * time.Minute,
-		},
+		client:    http.DefaultClient,
```

Comment:
I looked at https://golang.org/pkg/net/http/ but couldn't find the answer: Does the `DefaultClient` have no default timeout?